explore/src/main/scala/explore/services/OdbGroupApiImpl.scala

```diff
@@ -4,19 +4,26 @@
 package explore.services
 
 import cats.MonadThrow
+import cats.effect.Resource
 import cats.syntax.all.*
-import clue.FetchClient
+import clue.StreamingClient
 import clue.data.syntax.*
 import clue.syntax.*
 import eu.timepit.refined.types.numeric.NonNegShort
 import explore.model.Group
+import explore.model.ProgramTimeRange
 import lucuma.core.model.Program
 import lucuma.schemas.ObservationDB
 import lucuma.schemas.ObservationDB.Enums.Existence
 import lucuma.schemas.ObservationDB.Types.*
+import lucuma.schemas.odb.input.*
+import org.typelevel.log4cats.Logger
 import queries.common.GroupQueriesGQL.*
+import queries.common.ProgramQueriesGQL.GroupEditSubscription
+import queries.common.ProgramQueriesGQL.ProgramGroupsQuery
+import queries.common.ProgramSummaryQueriesGQL.GroupTimeRangeQuery
 
-trait OdbGroupApiImpl[F[_]: MonadThrow](using FetchClient[F, ObservationDB]):
+trait OdbGroupApiImpl[F[_]: MonadThrow](using StreamingClient[F, ObservationDB], Logger[F]):
```

Comment:
There is a new ObsCalcSubscription in GroupQueriesGQL used in ProgramCacheController